lib/archethic/contracts/interpreter/library.ex

```diff
@@ -243,4 +221,35 @@ defmodule Archethic.Contracts.Interpreter.Library do
       bin
     end
   end
+
+  @doc """
+  Get the genesis address of the chain
+  """
+  @spec get_genesis_address(binary()) ::
+          binary()
+  def get_genesis_address(address) do
+    addr_bin = decode_binary(address)
+    nodes = Election.chain_storage_nodes(address, P2P.authorized_and_available_nodes())
+
+    case TransactionChain.fetch_genesis_address_remotely(addr_bin, nodes) do
+      {:error, _} -> "error"
```

Comment:
I think we prefer the let it crash behavior when an error occurs. So maybe using `raise Atom.to_string(reason)`